lib/bundler/cli/update.rb

```diff
@@ -37,6 +37,9 @@ def run
         Bundler.definition(:gems => gems, :sources => sources, :ruby => options[:ruby])
       end
 
+      patch_level = [:major, :minor, :patch].detect { |v| options.keys.include?(v.to_s) }
```

Comment:
should favor left to right (maj, min, pat) ... specifying multiple wouldn't make sense internally. dunno if it should favor the last one specified by the user? e.g. `--minor --patch` => `--patch` vs. `--patch --minor` => `--minor`